VimR/VimR/States.swift

```diff
@@ -315,6 +317,7 @@ extension MainWindow {
 
     var appearance = AppearanceState.default
     var useInteractiveZsh = false
+    var nvimBinary: String = ""
```

Comment:
Wouldn't `String?` be more proper?

Reply:
It seems `String?` will result in `Attempt to insert non-property list object` exception. So, only `String` would work.

Reply:
ok, I put that back as was.